docs/tutorial/upload_publish_api.md

```diff
@@ -0,0 +1,748 @@
+<!-- markdownlint-disable MD013 -->
+<!-- Disabled MD013: long lines are needed in this tutorial -->
+
+# Upload and publish data using the NOMAD API
```

Comment:
I think what you did here is nice. One thought I had is that we have now semi-duplicated content 1. here, 2. in the nomad-utility-workflow docs, and 3. in the workflow and project management tutorial which is partially overlapping. This makes maintenance quite difficult. 

I do not think this affects your PR, just something general to think about in terms of which content we put where.